spec/rubocop/cop/style/redundant_string_escape_spec.rb

```diff
@@ -279,6 +279,14 @@ def wrap(contents)
     end
   end
 
+  context 'when string literal `?a`' do
```

Comment:
I guess that's not specific to `?a`, but applies to any char literal, right? Might be best to update the description accordingly.